src/js/view/controls/components/menu/settings-menu.js

```diff
@@ -314,6 +315,12 @@ class SettingsMenu extends Menu {
             gearButton.setAttribute('aria-expanded', true);
         }
 
+        if (getBreakpoint(this.model.get('containerWidth')) === 0 || getBreakpoint(this.model.get('containerWidth')) === 1) {
+            const mediaState = this.model.get('state')
```

Comment:
A little redundant since it's only used once below. Might be better to just use this.mediaStateWhenOpened = this.model.get('state');